actions/login.go

```diff
@@ -0,0 +1,74 @@
+package actions
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/cpjudge/cpjudge_webserver/models"
+	"github.com/gobuffalo/buffalo"
+	"github.com/gobuffalo/pop"
+	"golang.org/x/crypto/bcrypt"
+)
+
+// SigninHandler : Handles signin.
+func SigninHandler(c buffalo.Context) error {
+	fmt.Println("GET params were:", c.Request().URL.Query())
+	users := []models.User{}
+	err := models.DB.All(&users)
+	if err != nil {
+		fmt.Println(err)
+	}
+	contests := []models.Contest{}
+	err = models.DB.All(&contests)
+	if err != nil {
+		fmt.Println(err)
+	}
+	hosts := []models.Host{}
+	err = models.DB.All(&hosts)
+	if err != nil {
+		fmt.Println(err)
+	}
+	questions := []models.Question{}
+	err = models.DB.All(&questions)
+	if err != nil {
+		fmt.Println(err)
+	}
+	fmt.Println(questions)
+	fmt.Println(users)
+	fmt.Println(contests)
+	fmt.Println(hosts)
```

Comment:
Debugging attempts to be removed. We may need to use Logger for printing suitable logs of different levels.